packages/server/modules/fileuploads/services/resultListener.ts

```diff
@@ -63,6 +67,14 @@ export const onFileImportProcessedFactory =
     }
 
     if (isNewBranch) {
+      // Do relevant post-model-creation activities (e.g. set description, if any)
+      if (branch && upload.metadata?.description?.length) {
```

Comment:
easier to do this in the result handler than to implement this in every single file upload mechanism